src/site/content/en/fast/minify-css/index.md

```diff
@@ -0,0 +1,184 @@
+1. Once in the cloned project, click **Tools** (in the lower-left corner of the edit view), then select **Console**. The Glitch console opens in a new browser tab.
+
+To minify the resulting CSS, you’ll use the [optimize-css-assets-webpack-plugin](https://github.com/NMFR/optimize-css-assets-webpack-plugin): 
+
+1. In Glitch console, run `npm install --save-dev optimize-css-assets-webpack-plugin`.
+1. Run `refresh`, so the changes are synchronized with the Glitch editor.
+
+Next, go back to the Glitch editor, open the **webpack.config.js** file, and make the following modifications:
+
+Load the module at the beginning of the file:
+```
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+```
+
+Then, pass an instance of the plugin to the **plugins** array:
+```
+  plugins: [
+    new HtmlWebpackPlugin({template: "./src/index.html"}),
+    new MiniCssExtractPlugin({filename: "[name].css"}),
+    new FixStyleOnlyEntriesPlugin(),
+    new OptimizeCSSAssetsPlugin({})
+  ]
+```
+After making the changes a rebuild of the project will be triggered.
+This is how the resulting **webpack.config.js** will look like:
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
+    src="https://glitch.com/embed/#!/embed/fav-kitties-animated-min?path=webpack.config.js&previewSize=0"
+    alt="fav-kitties-animated-min on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+Next, you'll check the result of this optimization with performance tools.
+
+## Verify
+
+Click **Show live** and then, **In New Window**, to open your optimized app in a new window. If you got lost in any previous step, you can click [here](https://fav-kitties-animated-min.glitch.me/), to open an optimized version of the site.
+
+To inspect the size and content of the files:
+
+1. Open DevTools.
+1. Click in the **Network** panel and filter for “CSS”.
+1. Make sure "Disable Cache" is checked and reload the page.
+
+<img class="screenshot" width="700px" height="120px" src="./cdt-css-optimized.png" alt="DevTools CSS unoptimized response">
+
+You can inspect these files, and see that the new versions don't contain any whitespaces. Both files are much smaller, in particular, the [animate.css](http://fav-kitties-animated-min.glitch.me/animate.css) has been reduced in **~26%**, saving **~20KB**!
+
+As a final step:
+
+1. Open [Lighthouse](https://web.dev/measure). 
+1. Enter the URL of the optimized site.
+1. Click **View report**.
+1. Click on **Performance** and find the **Opportunities** section.
+
+The report doesn't show "Minify CSS" as "Opportunity" anymore, and has now moved to "Passed Audits" section:
+
+<img class="screenshot" width="700px" height="150px" src="./lighthouse-optimized.png" alt="Lighthouse Passed Audits for optimized page.">
```

Comment:
```suggestion
<img class="w-screenshot" width="700px" height="150px" src="./lighthouse-optimized.png" alt="Lighthouse Passed Audits for optimized page.">
```

Reply:
Btw for any image that is _almost_ centered but not quite, you can use the `w-figure--center` class. It's shown off in this section: https://web-dev-staging.appspot.com/sandbox/#images